src/main/java/com/commercetools/sync/commons/utils/ClientConfigurationUtils.java

```diff
@@ -14,6 +16,49 @@
 
 public final class ClientConfigurationUtils {
 
+  public static final String CLIENT_VERSION_V1 = "CLIENT_VERSION_V1";
+  public static final String CLIENT_VERSION_V2 = "CLIENT_VERSION_V2";
```

Comment:
Why do you need these?

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/859/commits/40ad6f992c7a67629613fee9de935190532e65f2